src/main/java/uk/gov/hmcts/reform/wataskmanagementapi/cft/query/RoleAssignmentFilter.java

```diff
@@ -204,29 +209,38 @@ private static Predicate mapSecurityClassification(
         CriteriaBuilder builder,
         RoleAssignment roleAssignment) {
         final Classification classification = roleAssignment.getClassification();
-        if (classification.equals(Classification.RESTRICTED)) {
-            return builder.in(root.get(SECURITY_CLASSIFICATION_COLUMN)).value(
-                List.of(
-                    SecurityClassification.RESTRICTED,
-                    SecurityClassification.PRIVATE,
-                    SecurityClassification.PUBLIC
-                )
-            );
-
-        } else if (classification.equals(Classification.PRIVATE)) {
-            return builder.in(root.get(SECURITY_CLASSIFICATION_COLUMN)).value(
-                List.of(
-                    SecurityClassification.PRIVATE,
-                    SecurityClassification.PUBLIC
-                )
-            );
-        } else {
-            return builder.in(root.get(SECURITY_CLASSIFICATION_COLUMN)).value(
-                List.of(
-                    SecurityClassification.PUBLIC
-                )
-            );
+        if (classification != null) {
```

Comment:
if the classification attribute is not provided should this fall under the default clause and search for everything PUBLIC/PRIVATE/RESTRICTED ?

Reply:
TBH, am not sure what the business requirement is if its not provided. We have not tested though so far.